apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue

```diff
@@ -73,39 +75,52 @@ watch(() => props.inherit, () => {
-                                   :invalid="!!state.errorMessage"
-                                   @update:selected="handleUpdateSelected"
-                                   @update:visible-menu="handleUpdateVisibleMenu"
+            <div>
+                <div v-if="props.label !== 'Cost Tag' || props.label !== 'Cost Additional Info'"
+                     class="select-form-wrapper"
                 >
-                    <template #dropdown-button>
-                        <div v-if="!props.nonInheritable && props.inherit"
-                             class="dropdown-inner"
-                        >
-                            <span class="item-label">{{ props.selected?.[0]?.label }}</span>
-                            <span class="suffix-text">{{ $t('DASHBOARDS.CUSTOMIZE.ADD_WIDGET.FROM_DASHBOARD') }}</span>
-                        </div>
-                    </template>
-                </p-select-dropdown>
-                <p-icon-button v-if="props.deletable"
-                               class="delete-button"
-                               shape="square"
-                               style-type="negative-secondary"
-                               name="ic_delete"
-                               @click="emit('delete')"
+                    <p-select-dropdown use-fixed-menu-style
+                                       :is-filterable="!props.inherit"
+                                       :multi-selectable="props.selectionType === 'MULTI'"
+                                       :handler="props.menuHandlers"
+                                       :selected="props.selected"
+                                       :invalid="!!state.errorMessage"
+                                       @update:selected="handleUpdateSelected"
+                                       @update:visible-menu="handleUpdateVisibleMenu"
+                    >
+                        <template #dropdown-button>
+                            <div v-if="props.inheritanceMode !== 'NONE' && props.inherit"
+                                 class="dropdown-inner"
+                            >
+                                <span class="item-label">{{ props.selected?.[0]?.label }}</span>
+                                <span class="suffix-text">{{ $t('DASHBOARDS.CUSTOMIZE.ADD_WIDGET.FROM_DASHBOARD') }}</span>
+                            </div>
+                        </template>
+                    </p-select-dropdown>
+                    <p-icon-button v-if="props.deletable"
+                                   class="delete-button"
+                                   shape="square"
+                                   style-type="negative-secondary"
+                                   name="ic_delete"
+                                   @click="emit('delete')"
+                    />
+                </div>
+                <!-- TODO: will be updated-->
+                <dashboard-cost-widget-value-option-dropdown
```

Comment:
question: Is this module only usable in the **cost** widget?🧐

Reply:
This component is named for the cost-tag and cost-additional-info options!